ckanext/switzerland/validators.py

```diff
@@ -249,7 +249,7 @@ def multiple_text_output(value):
     """
     Return stored json representation as a list
     """
-    return parse_json(value, default_value=[value])
+    return parse_json(value)
```

Comment:
This is what was causing ever greater layers of `[]` in fields that were mysteriously removed during validation. At first the field was empty, then it `[]`, then `[[]]`, then `[[[]]]`...